onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -3273,11 +3273,7 @@ void RegisterContribSchemas() {
            "tensor(float16)",
            "tensor(float)",
            "tensor(double)"},
-          "Constrain input and output types.")
-      .TypeAndShapeInferenceFunction([](ONNX_NAMESPACE::InferenceContext& ctx) {
```

Comment:
this part is no longer needed?

Reply:
Yes. We shouldn't assume the output data type same with any input data type.